onnx/defs/schema.h

```diff
@@ -28,11 +28,17 @@ namespace ONNX_NAMESPACE {
 
 struct FunctionBodyBuildContext {
   virtual const AttributeProto* getAttribute(const std::string& name) const = 0;
+
   virtual bool hasInput(int inputIndex) const = 0;
-  virtual bool hasOutput(int inputIndex) const = 0;
+  virtual bool hasOutput(int iutputIndex) const = 0;
+
+  virtual std::vector<std::string> getInputs() const = 0;
```

Comment:
Not sure why we need this?

Reply:
I removed this, and switched to using formal inputs/outputs, as suggested below.